ukrainian/fire/fire.md

```diff
@@ -1,2 +1,6 @@
 ﻿Смажений зефір чудово пахне, але чи варто його пробувати? Може він не їстівний?
-Варто спробувати, мабуть.
\ No newline at end of file
+Варто спробувати, мабуть.
+
+[Розженуся та спробую прорватися крізь одну з стін](break-through-wall/break-through-wall.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.